components/rewards/RewardTab.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {
+  RewardTabContainer,
+  RewardDescriptionContainer,
+  StarIcon
+} from '../../styled/rewards';
+import { Subhead, Caption } from '../BaseComponents';
+import { FontAwesome5 } from '@expo/vector-icons';
+import Colors from '../../assets/Colors';
+
+class RewardTab extends React.Component {
```

Comment:
Can you rename to RewardCard? 'Tab' seems confusing because this is all in what we'd call rewards tab.